src/libs/PolicyUtils.ts

```diff
@@ -461,6 +461,69 @@ function getXeroBankAccountsWithDefaultSelect(policy: Policy | undefined, select
     }));
 }
 
+function getNetSuiteVendorOptions(policy: Policy | undefined, selectedVendorId: string | undefined): SelectorType[] {
+    const vendors = policy?.connections?.netsuite.options.data.vendors ?? [];
```

Comment:
I am not sure whether we need safe access. 
```suggestion
    const vendors = policy?.connections?.netsuite?.options?.data?.vendors ?? [];
```